source/frontend/src/features/disposition/list/styles.ts

```diff
@@ -1,47 +0,0 @@
-import { Col } from 'react-bootstrap';
-import styled from 'styled-components';
-
-import { InlineForm, InlineInput } from '@/components/common/form/styles';
-import { Scrollable as ScrollableBase } from '@/components/common/Scrollable/Scrollable';
-
-export const ListPage = styled.div`
```

Comment:
This is the most opinionated part of this change - we essentially had this exact file copied to every list view in the system. I've removed these duplicates and replaced with a single set of common styles for list views.